OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -36,23 +40,484 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            </incipit>
+                                <note>The name of an individual, probably the owner of the manuscript, is mentioned in the supplication formula and partially effaced.
+                                Only 'Walda' is legible.
+                                </note>
+                            <incipit xml:lang="gez">
+                                <locus from="25va" to="25vb"/>
+                                ወያመጽኡ፡ መሥዋዕተ፡ ወቍርባነ፡ መሐና፡ ወስ<cb n="b"/>ሂነ፡ ወያመጽኡ፡ አምኃ፡ በአኰቴት፡ ለቤተ፡ እግዚአብሔር፡ ወእመሰ፡ ኢሰማዕክሙ፡ 
+                                ወአቅደምክሙ፡ ዕለተ፡ ሰንበት፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus from="25vb" to="26ra"/>
+                                <hi rend="rubric">እስመ፡</hi> ውእቱ፡ ዘበነቢያት፡ አስ<pb n="26r"/><cb n="a"/>ተንፈሰ፡ ሰንበታቲሁ፡ ቀደሰ፡ ከመ፡ ይኩነነ፡ መርሳ። ወዘበልየ፡ 
+                                ሐደሶ፡ ዘበመስቀሉ፡ ረከብነ፡ ሞገሰ።
+                            </explicit>
+                        </msItem>
+                            <msItem xml:id="ms_i1.15">
+                            <locus from="26ra" to="26va"/>
+                            <title><ref type="work" corresp="LIT1019Actsof.19.8-10">Acts 19,8–10</ref></title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez" type="inscription">
+                                <locus target="#26ra"/>
+                                <hi rend="rubric">ግብ</hi>ረ<hi rend="rubric">፡ ሐዋርያት፡</hi> 
+                            </incipit>
+                            <incipit xml:lang="gez">
+                                <locus from="26ra" to="26rb"/>
+                                ወቦአ፡ ጳውሎስ፡ ምኵራበ፡ ወነገረ፡ ገዓደ። ወነበረ፡ ፫ አውራኃ፡ ይትዋቀሶሙ፡ ወየአምኖሙ፡ በእንተ፡ መንግሥተ፡ እግዚአብሔር፡ ወቦ፡ እለ፡ ክህዱ፡ 
+                                ወ<cb n="b"/>ተንሥኡ፡ ወአሕሰሙ፡ ዲበ፡ ዘእግዚአብሒር፡ ትምሕርት።
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus target="#26va"/>
+                                <hi rend="rubric">ልዑል፡</hi> እምልዑላን፡ እኩት፡ በቅዱሳን፡ ዘለከ፡ ይትቄረብ፡ መዓዛ፡ ዕጣን፡ እምትጉሃን፡ ሐዋርያት፡ ይሴብሑከ፡ እለ፡ ተቀይሱ፡ 
+                                በሥርዓትከ፡ ተንሥ፡ በ፡ ስ፡ ለ፡ ም፡ መ፡ በሌሊት፡ አንሥኡ፡ በል፡ ቁሙ፡ አ፡ በል፡ 
+                            </explicit>
+                        </msItem>
+                            <msItem xml:id="ms_i1.16">
+                            <locus from="26va" to="28vb"/>
+                            <title><ref type="work" corresp="LIT1693John.5.1-18">John 5,1–18</ref></title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez" type="inscription">
+                                <locus target="#26va"/>
+                                <hi rend="rubric">ወንጌል፡ ዘዮሐንስ፡</hi> 
+                            </incipit>
+                            <incipit xml:lang="gez">
+                                <locus target="#26vb"/>
```

Comment:
26vb correct (or 26va)?

Reply:
the title ( `<incipit xml:lang="gez" type="inscription">`) is on fol. 26va, the proper incipit on fol. 26vb.